apps/qubit/modules/search/actions/advancedAction.class.php

```diff
@@ -501,6 +572,12 @@ public function execute($request)
       }
     }
 
+    // Process date range
+    if (null !== $criterias = $this->getDateRangeQuery($this->request['sd'], $this->request['ed']))
```

Comment:
you can just name this variable 'criteria', since it's already plural (criterion is for singular)